qcodes/dataset/plotting.py

```diff
@@ -90,7 +90,33 @@ def plot_by_id(run_id: int,
                                                  Number]] = None,
                **kwargs) -> AxesTupleList:
     """
-    Construct all plots for a given run
+    Construct all plots for a given `run_id`. All other arguments are forwarded
+    to :meth:`.plot_dataset` See this for more details.
+    """
+
+    dataset = load_by_id(run_id)
+    return plot_dataset(dataset,
+                        axes,
+                        colorbars,
+                        rescale_axes,
+                        auto_color_scale,
+                        cutoff_percentile,
+                        **kwargs)
+
+
+def plot_dataset(dataset: DataSet,
```

Comment:
shall this also be a method on the DataSet for convenience?

Reply:
As agreed offline we will leave this is as now